pkgs/development/tools/language-servers/fennel-ls/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchgit, lua, luaPackages }:
+stdenv.mkDerivation rec {
+  pname = "fennel-ls";
+  version = "0.1.0";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~xerool/fennel-ls/";
+    rev = version;
+    sha256 = "sha256-RW3WFJGwascD4YnnrAm/2LFnVigzgtfzVubLMDW9J5s=";
+  };
+
+  buildInputs = [ lua luaPackages.fennel];
+  makeFlags = [ "PREFIX=$(out)" ];
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "fennel language server";
```

I think this could need a small touch. How about something like this (taken from the repo)?

```suggestion
    description = "A language server for intelligent editing of the Fennel Programming Language";
```